### PR TITLE
[UI] toast implementation

### PR DESCRIPTION
**Pull request by peak3d**
- Toast implementation for wallet and transactions actions
- refresh addresses for rinkeby testnet

### CODE REVIEW

File: contracts/src/token/Token.sol [hunk 1]
```diff
@@ -111,7 +111,7 @@ contract WowsToken is ERC20Capped, AccessControl, IRewardHandler, AddressBook {
     _mint(__teamWallet, 7500 * 1e18);
     _teamWallet = __teamWallet;
 
-    // Multi-sig teamwallet has initial admin rights, eg for adding minters
+    // Multi-sig marketing wallet get admin rights
```
Comment (juztamau5) [inline contracts/src/token/Token.sol]: grammar suggestion: `gets`

File: src/locales/en_US/translation.json [hunk 2]
```diff
@@ -4,6 +4,13 @@
   "header": {
     "connectWallet": "CONNECT WALLET"
   },
+  "toast": {
+    "walletConnected": "Wallet connected",
+    "walletDisconnected": "Wallet disconnected",
+    "transactionStarted": "Transaction started...",
+    "transactionMined": "Transaction processing...",
+    "transactionFinished": "Transaction succeeded..."
```
Comment (juztamau5) [inline src/locales/en_US/translation.json]: "finished" mark the end of an event, not an ongoing one, so this line could be

```
    "transactionFinished": "Transaction succeeded"
```

**Comment by juztamau5** [comment]
LGTM. I'll runtime test soon.

File: contracts/src/token/Token.sol (hunk 1 above)
Comment (peak3d) [reply contracts/src/token/Token.sol]: thx

File: src/locales/en_US/translation.json (hunk 2 above)
Comment (peak3d) [reply src/locales/en_US/translation.json]: yes, already fixed here

Comment (juztamau5) [reply src/locales/en_US/translation.json]: doesn't look fixed when I CTRL+SHIFT+F5 refresh, but I'll take your word for it

**Review by juztamau5 — APPROVED** [review]
Looks good. Don't wait for me to runtime test, I'll get around to it tonight or this weekend and follow up with any feedback.

File: src/locales/en_US/translation.json (hunk 2 above)
Comment (peak3d) [reply src/locales/en_US/translation.json]: image

Comment (juztamau5) [reply src/locales/en_US/translation.json]: huh, github isn't updating for the second commit, it shows up [here](https://github.com/wolvesofwallstreet/wolves.finance/commit/94b3cc570a0bc8a851968e901548814567b3e2b7#diff-1d05998577b8b78bb4c40767b2b6bb40eccebc023376eb214a987788a7ce6ec6R12) but not in the files changed

**Pull request merged**